LocalMultiplayerAgent/MultiplayerSettings.json

```diff
@@ -1,51 +1,52 @@
 {
-    "RunContainer": false,
-    "OutputFolder": "",
-    "NumHeartBeatsForActivateResponse": 10,
-    "NumHeartBeatsForTerminateResponse": 60,
-    "AgentListeningPort": 56001,
-    "AssetDetails": [
-        {
-            "MountPath": "C:\\Assets",
-            "LocalFilePath": "<path_to_game_server_package>"
+  "RunContainer": false,
+  "forcePullFromAcrOnLinuxContainersOnWindows": false,
```

Comment:
maybe we don't need to add it here but add it on the MultiplayerSettingsLinuxContainersOnWindowsSample.json? So only users that want to run Linux containers on Windows can use it?

Reply:
moved ! :)